LanguageExt.Core/DataTypes/Recursive/Recursive.Prelude.cs

```diff
@@ -0,0 +1,451 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using LanguageExt.DataTypes.Recursive;
+
+namespace LanguageExt
+{
+    public static partial class Prelude
+    {
+
+        /// <summary>
+        /// Returns the base of the recursion
+        /// </summary>
+        /// <typeparam name="A"></typeparam>
+        /// <param name="t"></param>
+        /// <returns></returns>
+        public static Recursive<A> Return<A>(A a) => new Recursive<A>(a);
```

Comment:
Technically, `Yield` is also a noun (says the guy doing tons of analysis on corn yields). For a less _dangerous_ noun, perhaps `Outcome`, `Issue`, `Product` or `Harvest` (okay, maybe a bit corny)